tests/x11/remote_desktop/vino_screensharing_available.pm

```diff
@@ -35,18 +34,24 @@ sub run {
     # It may happen that the screen sharing is not available
     assert_screen [qw(with_screensharing without_screensharing)];
     if (match_has_tag 'without_screensharing') {
-        record_info 'vino missing', 'After the installation the screen sharing is not available - vino is missing and we need to install it now.';
-        send_key 'ctrl-q';
-
-        # Install the vino package which is probably the case of missing screen sharing option
-        ensure_installed 'vino';
+        if (is_sle("=15-sp4")) {
+            record_info 'gnome-remote-desktop', 'After the installation the screen sharing is not available - gnome-remote-dekstop is missing and we need to install it now.';
+            send_key 'ctrl-q';
+            # Install the vino package which is probably the case of missing screen sharing option
+            ensure_installed 'gnome-remote-desktop';
+        }
+        else {
+            record_info 'vino missing', 'After the installation the screen sharing is not available - vino is missing and we need to install it now.';
+            send_key 'ctrl-q';
 
+            # Install the vino package which is probably the case of missing screen sharing option
+            ensure_installed 'vino';
```

Comment:
Please add a comment on why vino is still tested, i.e (Or a link where this has been mentioned or discusse)